frontend/scripts/analytics/profile-node.events.js

```diff
@@ -0,0 +1,32 @@
+import { WIDGETS__SET_ENDPOINT_DATA } from 'react-components/widgets/widgets.actions';
+import { PROFILE_NODE__ON_PDF_DOWNLOAD } from 'react-components/profile-node/profile-node.actions';
+import { GET_NODE_SUMMARY_URL } from 'utils/getURLFromParams';
+
+export default [
+  {
+    type: WIDGETS__SET_ENDPOINT_DATA,
+    action: 'Profile page visited',
+    category: 'profile',
+    getPayload(action, state) {
+      const { data } = action.payload;
+      const {
+        location: { query },
+        app: { contexts }
+      } = state;
+      const context = contexts.find(ctx => Number(query.contextId) === ctx.id);
+
+      // eslint-disable-next-line
+      return `${data?.jurisdiction_name} - ${data?.column_name} - ${context?.countryName} - ${
```

Comment:
Jurisdiction name seems to be undefined for http://localhost:8081/profile-actor?nodeId=24603&contextId=1&year=2017
{hitType: "event", eventCategory: "profile", eventAction: "Profile page visited", eventLabel: "undefined - IMPORTER - BRAZIL - SOY - 2017"}

Reply:
woah thanks nice catch!